draft-ietf-tls-tls13.md

```diff
@@ -5069,6 +5072,58 @@ lacking a "server_name" extension by terminating the connection with a
 "missing_extension" alert.
 
 
+## Protocol Invariants
+
+This section describes invariants that TLS endpoints and intermediaries MUST
+follow. It also applies to earlier versions, which assumed these rules but did
+not document them.
+
+TLS is designed to be securely and compatibly extensible. Newer clients or
+servers, when communicating with newer peers, should negotiate the most
```

Comment:
SHOULD. Also, I note you are being vague about "preferred". TLS has generally assumed that the server picks the *highest* version but otherwise has discretion about how to pick the rest of the params.